packages/lib/gpcircuits/scripts/parameters/paramGen.ts

```diff
@@ -0,0 +1,553 @@
+// This file is the source of truth for GPC production circuit configurations.
+//
+// Run `yarn gen-circuit-parameters && yarn gen-artifacts` after editing.
+import { ProtoPODGPCCircuitParams } from "../../src";
+
+// Circuit parameters are grouped by numbers of objects and arranged in
+// ascending order wrt. number of entries.
+export const PARAMS: ProtoPODGPCCircuitParams[] = [
+  /*
+   * 1 object
+   */
+  {
+    maxObjects: 1,
+    maxEntries: 10,
+    merkleMaxDepth: 5,
+    maxNumericValues: 0,
+    maxEntryInequalities: 0,
+    maxLists: 0,
+    maxListElements: 0,
+    maxTuples: 0,
+    tupleArity: 0,
+    includeOwnerV3: false,
+    includeOwnerV4: true
+  },
+  {
+    maxObjects: 1,
+    maxEntries: 10,
+    merkleMaxDepth: 5,
+    maxNumericValues: 5,
+    maxEntryInequalities: 5,
+    maxLists: 2,
+    maxListElements: 20,
+    maxTuples: 1,
+    tupleArity: 3,
+    includeOwnerV3: false,
+    includeOwnerV4: true
+  },
+  // Minimal POD ticket configuration.
+  {
+    maxObjects: 1,
+    maxEntries: 11,
+    merkleMaxDepth: 5,
+    maxNumericValues: 0,
+    maxEntryInequalities: 0,
+    maxLists: 0,
+    maxListElements: 0,
+    maxTuples: 0,
+    tupleArity: 0,
+    includeOwnerV3: false,
+    includeOwnerV4: true
+  },
+  // Smaller POD ticket configuration including tuple and list membership check.
+  {
+    maxObjects: 1,
+    maxEntries: 11,
+    merkleMaxDepth: 5,
+    maxNumericValues: 0,
+    maxEntryInequalities: 0,
+    maxLists: 1,
+    maxListElements: 50,
+    maxTuples: 1,
+    tupleArity: 3,
+    includeOwnerV3: false,
+    includeOwnerV4: true
+  },
+  // Bigger POD ticket configuration including tuple and list membership check.
+  {
+    maxObjects: 1,
+    maxEntries: 11,
+    merkleMaxDepth: 5,
+    maxNumericValues: 0,
+    maxEntryInequalities: 0,
+    maxLists: 1,
+    maxListElements: 200,
+    maxTuples: 1,
+    tupleArity: 3,
+    includeOwnerV3: false,
+    includeOwnerV4: true
+  },
+  {
+    maxObjects: 1,
+    maxEntries: 20,
+    merkleMaxDepth: 6,
+    maxNumericValues: 0,
+    maxEntryInequalities: 0,
+    maxLists: 0,
+    maxListElements: 0,
+    maxTuples: 0,
+    tupleArity: 0,
+    includeOwnerV3: false,
+    includeOwnerV4: true
+  },
+  {
+    maxObjects: 1,
+    maxEntries: 20,
+    merkleMaxDepth: 6,
+    maxNumericValues: 20,
```

Comment:
I think having as many NVs and EIs as there are entries is probably overkill.  A ratio for N entries of N/2 numerics and N/4 inequalities seems like a reason able guess.  I see you following similar ratios elsewhere.  Keeping in mind that each inequality needs 2 numeric inputs in most cases (assuming they're independenent).